src/store/models/network.ts

```diff
@@ -107,6 +108,12 @@ export interface NetworkModel {
     StoreInjections,
     RootModel
   >;
+  updateTaroBackendNode: Thunk<
+    NetworkModel,
+    { id: number; taroName: string; LNDName: string },
```

Comment:
`LNDName` should be camel case `lndName`. `LND` should only be all caps when displaying the product name to users. All caps should only be used in code when defining a type that should start with a capital letter (ex: `class LNDService`). Variable names should always start with lower-case, so `lnd` is correct.